### ISSUE TITLE
Delete doesn't (always ?) function as portrayed in README

### ISSUE THREAD

**Issue by BramVer** [maarten-dp/flask-restless-client#33]
Basing myself off of how the `README.md` shows the deletion process works:
> ## D is for ... delete
> cmptr = c.Computer.query.one()
> cmptr.delete()

**This is my code:**
```python

    # Get all events with state before date
    query = client.Event.query
    query = query.filter_by(state='IMPORTED')
    query = query.filter(client.Event.created_at >= date)

    events = query.all()

    # Delete all events
    for i, event in enumerate(events):
        event.delete()
```

**What I expected to happen:**
The loop to process each event and that each event-instance got deleted.

**What happened instead:**
AttributeError got thrown because the `event` instance does not possess a `delete` method to call.  

I got this to work by calling `client.delete(event)` on each instance instead.  
Not sure whether something got overwritten, or the documentation is wrong.

**Comment by maartendp** [comment]
Yes, the readme is insanely out of date, I'll try to make some time to update it.

**Comment by BramVer** [comment]
Alright, wanted to shine some light onto this to make sure I used it correctly :+1:
